### ISSUE TITLE
Could this be used with watchify?

### ISSUE THREAD

**Issue by pke** [arian/partition-bundle#23]
I have my bundle splitting setup running with 'browserify' like this:
'browserify -p [ partition-bundle --map partitions.json --output ./www --url www --main index ]' 

However, watchify demands and output file, which I dont have when using `partition-bundle`.

**Comment by juice49** [comment]
@pke Specifying any output file seems to work for me. The file gets created but is empty, and my partitioned files get created as expected.

I've specified `/dev/null` and this seems to work too. Can't say for sure if this is correct, but it (seems) to get the job done.

Example:

```
watchify -p [ partition-bundle --map mapping.json --output public/dist --url dist --main ./src/index ] -o /dev/null
```

**Comment by pke** [comment]
~~Thanks , will try it out.~~ It works, thanks!

Now I wonder about your partion params. Care to share your mapping.json?

Mine looks like this:

```
{
  "index.js":   [ { "require": "./src/index.js", "expose": "index"} ],
  "home.js":    [ { "require": "./src/Home",     "expose": "home" } ],
  "cities.js":  [ { "require": "./src/Cities",   "expose": "cities" } ],
  "about.js":   [ { "require": "./src/About",    "expose":"about" } ]
} 
```

and in the `index.js` I can then: `loadjs(["home"], (Home)`
Without specifying the expose argument in the mapping I would have to `loadjs(["home.js"])` which is uncommon in CommonJS require calls.

**Comment by juice49** [comment]
@pke Sorry, I didn't receive an email for your comment.

I'm afraid I don't have my mapping file anymore, last Saturday was the first time I experimented with code splitting in Browserify and I ended up switching to Substack's [factor-bundle](https://github.com/substack/factor-bundle) and Filament Group's [loadJS](https://github.com/filamentgroup/loadJS).

I was a bit confused at first, and picked partition-bundle because it features a loader. I didn't really consider the fact that with factor-bundle, the split code simply has to be loaded via a script tag (or AJAXed and evald) before using it like normal.

Looking at my factor-bundle code, I realise I actually deleted a bunch of the stuff I was playing with. I think this is roughly what I ended up with:

```
'use strict';

import load from 'fg-loadjs';
import test from './test'; // test module is split by factor-bundle

const isBrowser = typeof window !== 'undefined';

if(isBrowser) {
  // if we're in the browser, we first need to load the split file before using it
  load('./test.js', test);
} else {
  // if the code has not been split, i.e. if it's running in node, just use it normally
  test();
}
```

Apologies to the author for commenting this here. I'm sure partition-bundle has many use cases, but it just wasn't quite what I was looking for 😊.

@pke I hope this is useful. The code might not run, but I hope explains what I mean!
